src/System.Net.NetworkInformation/tests/FunctionalTests/FileUtil.cs

```diff
@@ -31,5 +33,20 @@ public static void NormalizeLineEndings(string source, string normalizedDest)
 
             File.WriteAllText(normalizedDest, contents);
         }
+
+        /// <summary>
+        /// Look for the given test file in the current directory as well as the "NetworkFiles" subdirectory
+        /// </summary>
+        /// <returns>The correct path to a valid test file</returns>
+        public static string TestFile(string source)
+        {
+            if (File.Exists(source))
```

Comment:
There should be braces surrounding the code in each of the 'if', 'else if', 'else' clauses as per coding guidelines